config/manager/manager.yaml

```diff
@@ -19,6 +19,8 @@ spec:
   replicas: 1
   template:
     metadata:
+      annotations:
+      kubectl.kubernetes.io/default-container: manager
```

Comment:
nice annotation!
but it looks like it's not indentend correctly?

Reply:
That's correct, I will change that.